tcms/static/js/utils.js

```diff
@@ -151,6 +151,9 @@ function treeViewBind() {
     // collapse all child rows
     $('.tree-list-view-pf').find(".list-group-item-container").addClass('hidden');
 
+    // unbind all previous click handlers
+    $('.list-group-item-header').off('click')
```

Comment:
:+1: I was desperately debugging the same problem few days ago as part of #1957 but managed to workaround by avoiding the double call to treeViewBind(). This is much better!